osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -106,6 +114,19 @@ public enum GlobalAction
         [Description("Toggle mute")]
         ToggleMute,
 
+        // Song select keybindings
+        [Description("Toggle mod selection overlay")]
+        ToggleModSelection,
+
+        [Description("Select a random beatmap")]
+        SelectNextRandom,
+
+        [Description("Select the last random beatmap")]
+        SelectPreviousRandom,
+
+        [Description("Toggle beatmap options overlay")]
+        ToggleBeatmapOptions,
```

Comment:
Adding these bindings in the middle like that will mess up all users' keybindings (like they did mine when I ran it). Should probably be at the end?